project/Dependencies.scala

```diff
@@ -29,7 +29,8 @@ object Dependencies {
     "com.miguelfonseca.completely" % "completely-core"            % "0.8.0",
     "org.jsoup"                    % "jsoup"                      % "1.13.1",
     "org.webjars.npm"              % "govuk-frontend"             % "3.11.0",
-    "com.openhtmltopdf"            % "openhtmltopdf-pdfbox"       % "1.0.10"
+    "com.openhtmltopdf"            % "openhtmltopdf-pdfbox"       % "1.0.10",
+    "ai.x"                        %% "play-json-extensions"       % "0.42.0"
```

Comment:
Do we need this exstension?

Reply:
`FormTemplate` case class has more than 22 members. This extension supports a format for case classes with more than 22 members.